honeybee/radiance/command/rcollate.py

```diff
@@ -0,0 +1,101 @@
+# coding=utf-8
+from commandBase import RadianceCommand
+import os
+import sys
+from ..datatype import *
+from ..datatype import RadInputStringFormatter as Rfmt
+
+
+class Rcollate(RadianceCommand):
+    u"""
+    rcollate - resize or transpose matrix data
+
+    Attributes:
+
```

Comment:
I suggest to write the documentation as you go. It will be much harder later if you don't do it right now.
Also adding some Usage code will be really helpful. Should we have a .usage method for all the classes? In that case will be forced to write a usage code which can be also used for testing.